src/index.html

```diff
@@ -87,23 +87,12 @@ <h1 class="block-title display-1">Falcon</h1>
         <p>A Big Thank You to Our Project Patrons</p>
 
         <p>
-            <a href="https://www.govcert.lu/" target="_blank"><img src="img/sponsors/govcert.png" width="220" alt="CERT Gouvernemental Luxembourg" ></a>
-        </p>
-
-        <p>
-            <a href="https://www.kontrolnaya-rabota.ru/s/" target="_blank"><img src="img/sponsors/rabota.jpg" height="30" alt="Examination RU" style="margin-right: 10px"></a>
-
-            <a href="https://www.youtube.com/c/ParisNakitaKejser" target="_blank"><img src="img/sponsors/paris.svg" height="30" alt="Paris Kejser" style="margin-right: 10px"></a>
+            <a href="https://www.govcert.lu/" target="_blank"><img src="img/sponsors/govcert.png" width="220" alt="CERT Gouvernemental Luxembourg" style="margin-right: 10px"></a>
 
-            <a href="https://www.algolia.com" target="_blank" style="margin-right: -15px"><img src="img/sponsors/algolia.svg" height="30" alt="Algolia"></a>
-
-            <a href="https://www.salesforce.com" target="_blank"><img src="img/sponsors/salesforce.svg" height="30" alt="Salesforce"></a>
-        </p>
-
-        <p>
-            <a href="https://www.misaka.io" target="_blank" style="margin-right: 10px"><img src="img/sponsors/misaka.svg" height="30" alt="Misaka Network" style="filter: invert(1)"></a>
+            <a href="https://sentry.io/" target="_blank"><img src="img/sponsors/sentry.svg" height="60" alt="Sentry"></a>
 
-            <a href="https://github.com/LikaloLLC" target="_blank" style="margin-right: 10px"><img src="img/sponsors/likalo.png" height="30" alt="Likalo"></a>
+            <!-- Just adding a hidden logo so that it is served on HTTP. -->
+            <img src="img/sponsors/sentry-dark.svg" height="60" alt="Sentry (dark)" style="display: none;">
```

Comment:
I don't follow what's the objective of this line

Reply:
I don't want this logotype to be displayed here, but I want it to be served on `/assets/sentry-dark.svg`, is there a better way?

Reply:
don't all logo files get copied to the asset folder?

Reply:
No, only these that are referenced from the code it seems.

Reply:
btw seems there's a way of changing the image depending on the theme: https://dev.to/ziratsu/switch-to-a-darker-image-when-on-dark-mode-2lkh

Reply:
> No, only these that are referenced from the code it seems.

oh strange. well let's keep it like this